src/idlcxx/tests/streamer_generator.c

```diff
@@ -2292,9 +2414,9 @@ CU_Test(streamer_generator, bounded_sequence)
   test_bounded_sequence();
 }
 
-CU_Test(streamer_generator, typedef_resolution)
+CU_Test(streamer_generator, typedef_resolution, .disabled =true)  //not working at the moment, therefore skipping
```

Comment:
This one actually might work right now(?)

Reply:
If you take the latest `idlcxx` branch from Cyclone DDS that is... :sweat_smile:

Reply:
I will check 👍

Reply:
Yep, passes now!
Thanks!